src/components/drop-zone/DropZone.tsx

```diff
@@ -120,19 +119,23 @@ function DropZoneContent(
               <p>Drop the files here</p>
             </div>
           ) : isContainer ? null : (
-            <p>Drag and drop your files here, or click to select files</p>
+            <p
+              css={css`
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                transform: translate(-50%, -50%);
+              `}
```

Comment:
Can you put this in a variable since it's a duplication of the css just above

Reply:
fixed thank you